src/helpers.js

```diff
@@ -202,19 +237,19 @@ function updateField(field, requiredFields, node, formValues, logic, config) {
     field.isVisible = true;
   }
 
-  const updateValues = (fieldValues) =>
-    Object.entries(fieldValues).forEach(([key, value]) => {
-      // some values (eg "schema") are a function, so we need to call it here
+  const updateAttributes = (fieldAttrs) => {
```

Comment:
nit: I renamed it from `updateValues` because I thought it was about the field value (eg has_pet: "yes")